tcms/settings/common.py

```diff
@@ -381,6 +381,7 @@
     'tcms.issuetracker.types.GitHub',
     'tcms.issuetracker.types.Gitlab',
     'tcms.issuetracker.types.Redmine',
+    'tcms.issuetracker.types.AzureDevops'
```

Comment:
after rename that would go to the top. We try to keep a somewhat alphabetic order here, although JIRA is an exception. Should be after Gitlab.